streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -371,51 +515,11 @@ public void testStateGlobalThreadClose() throws Exception {
         assertEquals(streams.state(), KafkaStreams.State.NOT_RUNNING);
     }
 
-    @Test
-    public void globalThreadShouldTimeoutWhenBrokerConnectionCannotBeEstablished() {
```

Comment:
These two tests are not really for KafkaStreams but for StreamThreads / GlobalStreamThread.

Reply:
Are these cases already covered by existing tests over there? I didn't see them moved as part of this PR.

Reply:
The first test is covered in `GlobalStreamThreadTest#shouldThrowStreamsExceptionOnStartupIfExceptionOccurred`: we call partitionsFor during the initialization of the global task, which would throw if the broker is not connected.

The second test is covered in `shouldNotThrowWhenPendingShutdownInRunOnce` and `shouldNotThrowWithoutPendingShutdownInRunOnce` of `StreamThreadTest` to me.

Reply:
sounds good! Thanks for the confirmation.